examples/async-portals/src/Counter.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+const RAND = Math.round(Math.random() * 1000)
+
+class Counter extends React.Component {
+  state = { count: 0 }
+  gen = 0
+
+  catch = 42
+
+  componentDidMount() {
+    this.setState({
+      count: RAND,
+    })
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval)
+  }
+
+  render() {
+    return (
+      <span>
+        +{this.state.count}:{this.gen++}
```

Comment:
Examples should reflect real examples, I think this is not a real example. I suggest to create a simple Portal example.

Reply:
Or an Async Portal but without tricky things.

Reply:
I am using them for debug purposes. They reflect only the pure idea of RHL - don't lose the state.
This example is a good one, cos it actually __not__ working for sync portal, as described above.